src/main/webapp/content/scss/themes/_dark-variables.scss

```diff
@@ -127,13 +127,11 @@ $neutral-dark-l-20: lighten($neutral-dark, 20%);
 
 // Footer
 $footer-bg: $neutral-dark;
-$footer-height-dev: 3rem;
-$footer-height-prod: 2rem;
+$header-footer-combined-height: 2rem;
 
 // Sidebar
 $sidebar-width: 255px;
-$sidebar-footer-height-dev: 104px;
-$sidebar-footer-height-prod: 88px;
+$sidebar-header-footer-combined-height: 88px;
```

Comment:
We ensured that the dev navbar and footer has the same height as for prod, right?